src/packages/aws-cognito/src/entities/graphql-entity.ts

```diff
@@ -26,12 +26,12 @@ export class CognitoUser extends GraphQLEntity<CognitoUserBackendEntity> {
 		);
 	}
 
-	@Field(() => String)
+	@Field(() => String, { nullable: true })
 	async userStatus(dataEntity: CognitoUserBackendEntity) {
```

Comment:
If we do this:

```suggestion
	async userStatus({ dataEntity }: CognitoUserBackendEntity) {
```
Then the next line can just be:
```typescript
return dataEntity.UserStatus;
```